tests/mlmodel_bedrock/_test_chat_completion.py

```diff
@@ -0,0 +1,139 @@
+chat_completion_payload_templates = {
+    "amazon.titan-text-express-v1": '{ "inputText": "%s", "textGenerationConfig": {"temperature": %f, "maxTokenCount": %d }}',
+    "ai21.j2-mid-v1": '{"prompt": "%s", "temperature": %f, "maxTokens": %d}',
+}
+
+chat_completion_expected_events = {
+    "amazon.titan-text-express-v1": [
+        (
+            {"type": "LlmChatCompletionSummary"},
+            {
+                "id": None,  # UUID that varies with each run
+                "appName": "Python Agent Test (mlmodel_bedrock)",
+                "conversation_id": "my-awesome-id",
+                "transaction_id": None,
+                "span_id": "span-id",
+                "trace_id": "trace-id",
+                "request_id": "660d4de9-6804-460e-8556-4ab2a019d1e3",
+                "api_key_last_four_digits": "CRET",
+                "duration": None,  # Response time varies each test run
+                "request.model": "amazon.titan-text-express-v1",
+                "response.model": "amazon.titan-text-express-v1",
+                "response.usage.completion_tokens": 55,
+                "response.usage.total_tokens": 67,
+                "response.usage.prompt_tokens": 12,
+                "request.temperature": 0.7,
+                "request.max_tokens": 100,
+                "response.choices.finish_reason": "FINISH",
+                "vendor": "bedrock",
+                "ingest_source": "Python",
+                "number_of_messages": 2,
+            },
+        ),
+        (
+            {"type": "LlmChatCompletionMessage"},
+            {
+                "id": None,  # UUID that varies with each run
+                "appName": "Python Agent Test (mlmodel_bedrock)",
+                "conversation_id": "my-awesome-id",
+                "request_id": "660d4de9-6804-460e-8556-4ab2a019d1e3",
+                "span_id": "span-id",
+                "trace_id": "trace-id",
+                "transaction_id": None,
+                "content": "What is 212 degrees Fahrenheit converted to Celsius?",
+                "role": "user",
+                "completion_id": None,
+                "sequence": 0,
+                "request.model": "amazon.titan-text-express-v1",
+                "response.model": "amazon.titan-text-express-v1",
+                "vendor": "bedrock",
+                "ingest_source": "Python",
```

Comment:
I'd say we shouldn't have the ingest_source on here since we have instrumentation.provider in ml_events but since the plan is to change back to custom events now with larger limits we'll need to add this back anyway so might as well just keep it here.